hibernate-core/src/main/java/org/hibernate/boot/internal/MetadataImpl.java

```diff
@@ -84,7 +85,7 @@
 	public MetadataImpl(
 			UUID uuid,
 			MetadataBuildingOptions metadataBuildingOptions,
-			TypeResolver typeResolver,
+			TypeConfiguration typeConfiguration,
```

Comment:
Is there a reason we can't just get the `TypeConfiguration` from the `BoootstrapContext` here?

Reply:
just an oversight. Thanks